optionals/compat_rhs_usf3/CfgVehicles.hpp

```diff
@@ -316,4 +318,57 @@ class CfgVehicles {
     class rhsusf_airforce_jetpilot: rhsusf_usmc_marpat_wd_rifleman_m4 {
         ace_gforcecoef = 0.55;
     };
+
+    class Items_base_F;
+    class ACE_Explosives_Place: Items_base_F {
+        class EventHandlers {
+            class CBA_Extended_EventHandlers;
+        };
+        class ACE_Actions {
+            class ACE_MainActions {
+                class ACE_SetTrigger;
+                class ACE_PickUp;
+            };
+        };
+    };
+
+    class ACE_Explosives_Place_rhsusf_m112_DemoCharge: ACE_Explosives_Place {
+        displayName = "USAF Demo Charge";
```

Comment:
Use their string key instead of the translated name, so this name shows up correctly in non English settings.